README.markdown

```diff
@@ -1,6 +1,6 @@
 # Fourth Wall
 
-[![Build Status](https://travis-ci.org/alphagov/fourth-wall.png?branch=gh-pages)](https://travis-ci.org/alphagov/fourth-wall?branch=gh-pages)
+[![Build Status](https://travis-ci.org/alphagov/fourth-wall.png)](https://travis-ci.org/alphagov/fourth-wall?branch=gh-pages)
```

Comment:
Aha, didn't realise it was `gh-pages` as default, rather than master.

Regardless, the `?branch=gh-pages` was breaking the build image. I'll cleanup the commit message, and remove the branch param on the href too.